telemetry/src/config.rs

```diff
@@ -16,8 +16,22 @@ pub struct Configuration {
     /// The url of the telemetry, e.g., ws://127.0.0.1:8001/submit
     #[config(serde_as_str)]
     pub url: Option<Url>,
+    /// The minimum period of time in seconds to wait before reconnecting
+    #[serde(default = "default_min_period")]
+    pub min_period: u64,
+    /// The maximum power of 2 that is used for increasing delay between reconnections
+    #[serde(default = "default_max_power")]
+    pub max_power: u64,
     /// The filepath that to write dev-telemetry to
     #[cfg(feature = "dev-telemetry")]
     #[config(serde_as_str)]
     pub file: Option<PathBuf>,
 }
+
```

Comment:
This might be better to move into `impl Default for RetryPeriod`.

Reply:
There is no Default for RetryPeriod.